src/app/modules/browse/browse.tpl.html

```diff
@@ -74,8 +74,6 @@ <h3 class="bookTitle">{{book.title}}</h3>
                             </li>
                         </ul>
                     </div>
-                    <!-- temporary affordance for trying out the new datagrid. Not the final design -->
-                    <!--form action="#datagrid"><input type="submit" value="Grid view"></form-->
```

Comment:
Is there some new way to launch it? I guess you can just enter the URL for now, but we will need something eventually...has JohnH specified what he wants?